src/misc/src/1DQuadrature.C

```diff
@@ -33,12 +33,10 @@ Base1DQuadrature::Base1DQuadrature(
   double minDomainValue,
   double maxDomainValue,
   unsigned int order)
-  :
+  : QuadratureBase(),
   m_minDomainValue(minDomainValue),
   m_maxDomainValue(maxDomainValue),
-  m_order         (order),
-  m_positions     (0),
-  m_weights       (0)
+  m_order         (order)
 {
   queso_require_less_msg(m_minDomainValue, m_maxDomainValue, "min >= max");
```

Comment:
Actually it appears in some cases you allow an infinite domain; `GaussianHermite1DQuadrature`.